test/Microsoft.AspNetCore.Hosting.Tests/WebHostTests.cs

```diff
@@ -23,40 +23,12 @@
 
 namespace Microsoft.AspNetCore.Hosting
 {
-    public class WebHostTests : IServer
+    public class WebHostTests
```

Comment:
Do you have any tests that call into the synchronous extension methods? Did you run into any issues with the xunit sync context?